build/package/Installer.DEB.targets

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Target Name="SetupDebProps"
-          DependsOnTargets="Init">
+      DependsOnTargets="Init">
```

Comment:
These kinds of changes shouldn't happen.  Lining the attributes up vertically is very common, and helps in reading these files.